package.json

```diff
@@ -69,6 +69,7 @@
     "express": "4.19.2",
     "express-async-errors": "3.1.1",
     "express-session": "1.18.0",
+    "fast-xml-parser": "^4.4.1",
```

Comment:
We don't use this package directly so we should probably find the package that uses it and upgrade that if we can - looking in package-lock it looks like `@aws-sdk/client-sts` uses this which itself is a dependency of `@aws-sdk/client-dynamodb` - can we try updating this to the latest?